docs/faq.md

```diff
@@ -279,7 +279,21 @@ python3 -m pip install twine
 python3 -m twine upload wheelhouse/*.whl
 ```
 
-### 'ImportError: DLL load failed: The specific module could not be found' error on Windows
+### macOS: Passing DYLD_LIBRARY_PATH to delocate
+
+macOS has built-in [System Integrity protections](https://developer.apple.com/library/archive/documentation/Security/Conceptual/System_Integrity_Protection_Guide/RuntimeProtections/RuntimeProtections.html) which limits the use of `DYLD_LIBRARY_PATH` and `LD_LIBRARY_PATH` so that it does not automatically pass to children processes. This means if you set `DYLD_LIBRARY_PATH` before running cibuildwheel, or even set it in `CIBW_ENVIRONMENT`, it will be stripped out of the environment before delocate is called.
+
+To work around this, use a different environment variable such as `REPAIR_LIBRARY_PATH` to store the library path, and set `DYLD_LIBRARY_PATH` in [`CIBW_REPAIR_WHEEL_COMMAND_MACOS`](https://cibuildwheel.readthedocs.io/en/stable/options/#repair-wheel-command), like this:
+
+```yaml
+CIBW_REPAIR_WHEEL_COMMAND_MACOS: >
+  DYLD_LIBRARY_PATH=$REPAIR_LIBRARY_PATH delocate-listdeps {wheel} &&
+  DYLD_LIBRARY_PATH=$REPAIR_LIBRARY_PATH delocate-wheel --require-archs {delocate_archs} -w {dest_dir} {wheel}
+```
```

Comment:
Can we do tabs and show the TOML version?

Reply:
yeah, that would be nice

Reply:
I've added them. Look okay to you?